### PR TITLE
Er

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,22 +10,28 @@ The dependency list is short: numpy and Cython (>= 0.21.1).
 Usage
```
Comment (matt-peters) [inline README.md]: Please add some context and details to this README:
- What is the main functionality in this repo?  Why should I care about it?
- Once we have learned a vocab how do we use it?  What are some interesting things we can use it to do?

Comment (matt-peters) [inline README.md]: Also it'd be great to document our assumptions around the set of valid n-grams: can't start/end with n-grams, each token must be a uni-gram, etc.

Comment (matt-peters) [inline README.md]: And also:
- the tokenizer
- anything a user needs to know about encoding, especially if they are implementing their own tokenizer

File: vocab/vocab.pyx [hunk 2]
```diff
@@ -32,23 +32,68 @@ def alpha_tokenize(s):
...
+        self._table = IndexLookupTable._build(counts,
+                                              self._table_size, self._power)
+
+    @staticmethod
+    def _build(counts, table_size, power):
+        """
+        Create a table using the vocabulary tokens counts, which can be used
+        for drawing random words in negative sampling routines (useful for
+        word2gauss.)
+        This function is called by the constructor if the build_table
```
Comment (matt-peters) [inline vocab/vocab.pyx]: I'm not sure what this comment means in this context (this class doesn't have a build_table option in the constructor).

File: vocab/vocab.pyx [hunk 3]
```diff
@@ -85,42 +129,7 @@ cdef class Vocabulary:
...
-        vocab_size = len(self.counts)
-        for tidx in xrange(table_size):
-            table[tidx] = widx
-            if 1.0 * tidx / table_size > d1:
-                widx += 1
-                d1 += self.counts[widx]**power / train_words_pow
-            if widx >= vocab_size:
-                widx = vocab_size - 1
-        return table
+        self._vocabptr = new Vocab(v, STOP_WORDS)
```
Comment (matt-peters) [inline vocab/vocab.pyx]: It'd be awesome to allow the user to specify the stop word list if they didn't want to use the default list.

File: vocab/vocab.pyx [hunk 4]
```diff
@@ -128,6 +137,55 @@ cdef class Vocabulary:
...
         return self._vocabptr.size()
 
+    def create(self, corpus, ngram_counts, keep_unigram_stopwords=True):
+        """
+        Create the vocabulary from the given corpus
+
+        corpus: an iterable that produces documents (as strings) from a corpus
+        ngram_counts: a list of tuples, one per ngram order that specifies the
+        maximum number of tokens to keep, the minimum count required
+        for a token, and a discounting
```
Comment (matt-peters) [inline vocab/vocab.pyx]: seems like this sentence was never finished.

**Comment by ghost** [comment]
@matt-peters

File: vocab/vocab.pyx (hunk 2 above)
Comment (ghost) [inline vocab/vocab.pyx]: To simplify the interface, I'd removed the option, but forgot to update the comment.

**Comment by matt-peters** [comment]
Looks great, thank you!

**Pull request merged**